hw2/css/style.css

```diff
@@ -0,0 +1,164 @@
+
+		.menu-top{
+			display: flex;
+			line-height: 40px;
+			width: 100%;
+			background-color: #fff;
+		    background-image: -webkit-linear-gradient(bottom, #444343 0%, #5e5e5e 100%);
+		    background-image: -o-linear-gradient(bottom, #444343 0%, #5e5e5e 100%);
+		    background-image: linear-gradient(to top, #444343 0%, #5e5e5e 100%);
+		}
+
+			.menu-top a{
+				text-shadow: 5px 5px 7px rgba(0, 0, 0, 0.8);
+				font-family: "Open Sans", "Fira Sans", Arial;
+				padding: 0 4%;
+				color: #fff;
+			}
+
+				.menu-top a:hover{
+					cursor: pointer;
+					background-color: #fff;
+				    background-image: -webkit-linear-gradient(bottom, #2e2e2e 0%, #4a4a4a 100%);
+				    background-image: -o-linear-gradient(bottom, #2e2e2e 0%, #4a4a4a 100%);
+				    background-image: linear-gradient(to top, #2e2e2e 0%, #4a4a4a 100%);
+				    outline: 1px solid #838383;
+				}
+
+		.main-content{
+			margin-top: 40px;
+		}
+
+			.main-content__menu{
+				display: inline-block;
+				width: 22.5%;
+				border: 1px solid #c8c8c8;
+			}
+
+				.main-content__menu>a{
+					display: block;
+					vertical-align: middle;
+					font-family: "Open Sans", "Fira Sans", Arial;
+					font-size: 13px;
+					font-weight: 400;
+					line-height: 37px;
+					padding-left: 22px;
+					color: #656363;
+					border-bottom: 1px solid #c8c8c8;
+					cursor: pointer;
+				}
+
+					.main-content__menu a:last-child{
+						border-bottom: none;
+					}
+
+					.main-content__menu a:hover{
+						color: #2c2c2c;
+						background-color: #f9f9f9;
+						padding-left: 32px;
+						cursor: pointer;
+					}
+
+			.main-content__product{
+				float: right;
+				width: 72%;
+			}
+
+				.main-content__product-img{
+					border: 1px solid #c8c8c8;
+					margin-bottom: 39px;
+					text-align: center;
+					padding: 2%;
+					width: 89%;
+					float: right;
+				}
+
+					.main-content__product-img>img{
+						width: 74%;
+					}
+
+				.main-content__description{
+					margin: 0 29px 0 24px;
+				}
+
+					.main-content__description>p{
+						color: #2c2c2c;
+						font-family: "Open Sans", "Fira Sans", Arial;
+						font-size: 13px;
+						font-weight: 400;
+						line-height: 21px;
+						padding: 5%;
+					}
+
+		footer{
+			position: relative;
+			margin-top: 60px;
+			font-size: 0;
+			color: #2c2c2c;
+			font-family: "Open Sans", "Fira Sans", Arial;
+			font-weight: 400;
+			background-image: -webkit-linear-gradient(bottom, #e1e1e1 0%, #e1e1e1 100%);
+			background-image: -o-linear-gradient(bottom, #e1e1e1 0%, #e1e1e1 100%);
+			background-image: linear-gradient(to top, #e1e1e1 0%, #e1e1e1 100%);
+		}
+
+			.footer__menu{
+				width: 70%;
+				display: inline-block;
+				line-height: 29px;
+			}
+
+				.footer__menu>a{
+					display: inline-block;
+					width: 14%;
+					vertical-align: middle;
+					font-size: 13px;
+					color: #000;
+					padding-left: 4%;
+				}
+
+					.footer__menu a:hover{
+						text-decoration: underline;
+					}
+
+			.copyrights{
+				position: absolute;
```

Comment:
bad solution. Don't use positions if you can use floats-inlines-flexes and so on. All these are a lot more flexible to use. For example if there will be more items in menu and they overflow copyright (of for example if translation of these items will be longer) then they could break you menu, but items still be clickable if you use float, but it will be not clickable if you use position and will stand with this block before menu-item